### PR TITLE
✏️Refactored docstrings to all be in RST format

### LINKED ISSUE #17: unify source code docstrings
Discuss with team what our python style standards. Add docstrings and comments to existing code that adhere to agreed upon coding standards.

### PR DESCRIPTION
**Pull request by gsantia**
edited docstrings to be in rst

closes #17

### CODE REVIEW

**Comment by gsantia** [comment]
sorry, forgot to rebase! :[

File: kf_lib_data_ingest/app/cli.py [hunk 1]
```diff
@@ -150,14 +150,10 @@ def ingest(
 ):
```
Comment (znatty22) [inline kf_lib_data_ingest/app/cli.py]: Can you keep the blob you removed and also keep what you added? I know thats redundant but we need that specially formatted argument description (using the `\b`) in order for the CLI to include it in its help text for positional arguments.

**Comment by znatty22** [comment]
Thank you for doing this! 

Couple requests:

1. Can you do rebase like we did on the `add-quickstart` branch and delete all of those old commits up to and including the `add quickstart d6667b8`?

2. Can you emojify your commit messages (can use git rebase and the `r` for reword)

3. In the PR title, can you prefix it with the pencil emoji (use actual emoji, not the `:pencil:` shortcode because GitHub won't render the shortcode as an emoji)

File: kf_lib_data_ingest/app/cli.py [hunk 2]
```diff
@@ -150,14 +150,10 @@ def ingest(
...
     """
     Run the Kids First data ingest pipeline.
-
-    \b
```
Comment (znatty22) [inline kf_lib_data_ingest/app/cli.py]: Can you undelete this section? We actually do need it bc the CLI renders it in its help text for positional arguments

File: kf_lib_data_ingest/app/cli.py [hunk 3]
```diff
@@ -217,14 +213,10 @@ def test(
     """
     Run the Kids First data ingest pipeline in dry_run mode (--dry_run=True)
     Used for testing ingest packages.
-
```
Comment (znatty22) [inline kf_lib_data_ingest/app/cli.py]: Same comment as above

**Comment by fiendish** [comment]
sad that we're not using google format, but does this close issue 17?

**Comment by fiendish** [comment]
closes #17

**Pull request merged**